.github/workflows/docker-ci.yml

```diff
@@ -0,0 +1,22 @@
+---
+name:
+  "Docker Image CI"
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Reminder for the future:  This was necessary because otherwise `actions/checkout@v2` will add a merge commit to the repo within the runner, and when `lint_git` runs, it will fail as the commit message is not compliance with the conventional commits format.